core/dslmcode/shared/drupal-7.x/modules/elmsln_contrib/cis_connector/features/elmsln_cdn/elmsln_cdn.module

```diff
@@ -3,3 +3,99 @@
  * @file
  * Drupal needs this blank file.
  */
+
+/**
+ * Implements hook_menu().
+ */
+  function elmsln_cdn_menu() {
+    // Admin UI.
+    $items['admin/config/development/elmsln_cdn'] = array(
+      'title'            => 'ELMSLN CDN',
+      'description'      => 'Configure ELMSLN CDN integration.',
+      'access arguments' => array(CDN_PERM_ADMIN),
+      'page callback'    => 'drupal_get_form',
+      'page arguments'   => array('elmsln_cdn_admin_general_settings_form'),
+      'type'             => MENU_NORMAL_ITEM,
+      'file'             => 'elmsln_cdn.admin.inc',
+    );
+
+    return $items;
+  }
+
+/**
+ * Implements hook_module_implements_alter().
+ */
+function elmsln_cdn_module_implements_alter(&$implementations, $hook) {
+  if ($hook == 'js_alter') {
+    $group = $implementations['elmsln_cdn'];
+    unset($implementations['elmsln_cdn']);
+    $implementations['elmsln_cdn'] = $group;
+  }
+}
+
+/**
+ * Implements hook_js_alter().
+ */
+function elmsln_cdn_js_alter(&$javascript) {
+  foreach ($javascript as $path => $values) {
+    if (strpos(strval($path), 'sites/all/libraries/') !== FALSE) { 
```

Comment:
probably can be expressed as `strpos(strval($path), 'sites/all/libraries/') === 0` as you are looking to see if its int he first position of the string and not "does it exist somewhere" which the literal might not return.